src/adhocracy_meinberlin/adhocracy_meinberlin/resources/subscriber.py

```diff
@@ -1,8 +1,68 @@
 """Initialize meinberlin ACM."""
 from pyramid.events import ApplicationCreated
+from pyramid.traversal import find_interface
+from pyramid.renderers import render
+
+from adhocracy_core.interfaces import IResourceCreatedAndAdded
 from adhocracy_core.authorization import set_acms_for_app_root
 from adhocracy_core.resources.root import root_acm
+from adhocracy_core.utils import get_sheet
 from adhocracy_meinberlin.resources.root import meinberlin_acm
+from adhocracy_meinberlin.resources.bplan import IProposalVersion
+from adhocracy_meinberlin.resources.bplan import IProposal
+from adhocracy_meinberlin import sheets
+from adhocracy_meinberlin import resources
+
+
+def _send_bplan_submission_confirmation_email_subscriber(event):
+    if not hasattr(event.registry, 'messenger'):  # ease testing
+        return
+    messenger = event.registry.messenger
+    proposal_version = event.object
+    proposal_item = find_interface(proposal_version, IProposal)
+    if not _bplan_proposal_has_been_created(proposal_item):
```

Comment:
I did not understand this at once,  maybe better name "is_proposal_creation_finished(proposal_version)"